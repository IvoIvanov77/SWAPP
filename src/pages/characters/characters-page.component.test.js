import React from 'react';
import { shallow } from 'enzyme';
import CharactersPage from './characters-page.component';

describe('CharactersPage component', () => {
  const wrapper = shallow(<CharactersPage />);

  it('should render CharactersPage component', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render characters list', () => {
    expect(wrapper.exists('CharactersListContainer')).toBe(true);
  });
});
