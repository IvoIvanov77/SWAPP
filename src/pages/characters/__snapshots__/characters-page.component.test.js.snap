// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharactersPage component should render CharactersPage component 1`] = `ShallowWrapper {}`;
